common/djangoapps/track/tests/test_contexts.py

```diff
@@ -6,6 +6,21 @@
 from common.djangoapps.track import contexts
 
 
+from django.conf import settings
+ENTERPRISE_ENABLED = 'enterprise' in settings.INSTALLED_APPS \
+                     and settings.FEATURES.get('ENABLE_ENTERPRISE_INTEGRATION', False)
+if ENTERPRISE_ENABLED:
```

Comment:
@rehanedly why we introduced `if` condition to import enterprise modules?

Reply:
There was an error of importing modules from partially installed module. That's why I have added this condition